front-end/src/app/shared/models/transaction-types/OPERATING_EXPENDITURE.model.ts

```diff
@@ -0,0 +1,23 @@
+import { LabelUtils } from 'app/shared/utils/label.utils';
+import { schema } from 'fecfile-validate/fecfile_validate_js/dist/COMMON_DISBURSEMENTS';
+import { AggregationGroups } from '../transaction.model';
+import { SchBTransaction, ScheduleBTransactionTypeLabels, ScheduleBTransactionTypes } from '../schb-transaction.model';
+import { SchbTransactionType } from './SchbTransactionType.model';
+import { STANDARD_CONTROLS, TransactionNavigationControls } from '../transaction-navigation-controls.model';
+import { ContactTypes } from '../contact.model';
+
+export class OPERATING_EXPENDITURE extends SchbTransactionType {
+  componentGroupId = 'H';
+  title = LabelUtils.get(ScheduleBTransactionTypeLabels, ScheduleBTransactionTypes.OPERATING_EXPENDITURE);
+  schema = schema;
+  override defaultContactTypeOption = ContactTypes.ORGANIZATION;
```

Comment:
I love that you pulled this property out of the options list.  This makes it explicit rather than a side effect